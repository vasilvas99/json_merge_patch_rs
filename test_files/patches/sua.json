{
    "name": "sua",
    "image": {
        "name": "ghcr.io/eclipse-leda/leda-contrib-self-update-agent/self-update-agent:build-20"
    },
    "mounts": [
        {
            "source": "/var/run/dbus/system_bus_socket",
            "destination": "/var/run/dbus/system_bus_socket",
            "propagation_mode": "shared"
        },
        {
            "source": "/data/selfupdates",
            "destination": "/RaucUpdate",
            "propagation_mode": "rprivate"
        }
    ],
    "host_config": {
        "extra_hosts": [
            "mosquitto:host_ip"
        ],
        "port_mappings": [
            {
                "protocol": "tcp",
                "container_port": 50052,
                "host_ip": "localhost",
                "host_port": 30052,
                "host_port_end": 30052
            }
        ]
    }
}
