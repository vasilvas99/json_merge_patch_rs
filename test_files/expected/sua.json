{
    "id": "",
    "name": "sua",
    "image": {
        "name": "ghcr.io/eclipse-leda/leda-contrib-self-update-agent/self-update-agent:build-20",
        "decrypt_config": null
    },
    "host_name": "",
    "domain_name": "",
    "resolv_conf_path": "",
    "hosts_path": "",
    "hostname_path": "",
    "mounts": [
        {
            "source": "/var/run/dbus/system_bus_socket",
            "destination": "/var/run/dbus/system_bus_socket",
            "propagation_mode": "shared"
        },
        {
            "source": "/data/selfupdates",
            "destination": "/RaucUpdate",
            "propagation_mode": "rprivate"
        }
    ],
    "hooks": [],
    "host_config": {
        "devices": [],
        "network_mode": "bridge",
        "privileged": false,
        "restart_policy": {
            "maximum_retry_count": 0,
            "retry_timeout": 0,
            "type": "unless-stopped"
        },
        "runtime": "io.containerd.runc.v2",
        "extra_hosts": [
            "mosquitto:host_ip"
        ],
        "port_mappings": [
            {
              "protocol": "tcp",
              "container_port": 50052,
              "host_ip": "localhost",
              "host_port": 30052,
              "host_port_end": 30052
            }
        ],
        "log_config": {
            "driver_config": {
                "type": "json-file",
                "max_files": 2,
                "max_size": "1M",
                "root_dir": ""
            },
            "mode_config": {
                "mode": "blocking",
                "max_buffer_size": ""
            }
        },
        "resources": null
    },
    "io_config": {
        "attach_stderr": false,
        "attach_stdin": false,
        "attach_stdout": false,
        "open_stdin": false,
        "stdin_once": false,
        "tty": false
    },
    "config": {
        "env": [],
        "cmd": []
    },
    "network_settings": null,
    "state": {
        "pid": -1,
        "started_at": "",
        "error": "",
        "exit_code": 0,
        "finished_at": "",
        "exited": false,
        "dead": false,
        "restarting": false,
        "paused": false,
        "running": false,
        "status": "",
        "oom_killed": false
    },
    "created": "",
    "manually_stopped": false,
    "restart_count": 0
}